addons/sale/models/product_product.py

```diff
@@ -59,29 +59,33 @@ def _get_combination_info_variant(self, add_qty=1, pricelist=False, parent_combi
 class ProductAttribute(models.Model):
     _inherit = "product.attribute"
 
-    # YTI FIX ME: PLEASE RENAME ME INTO attribute_type
-    type = fields.Selection([
+    display_type = fields.Selection([
         ('radio', 'Radio'),
         ('select', 'Select'),
-        ('color', 'Color')], default='radio', required=True)
+        ('color', 'Color')], default='radio', required=True, help="The display type used in the Product Configurator")
 
 
 class ProductAttributeValue(models.Model):
     _inherit = "product.attribute.value"
 
+    # Hide the entire column if not color
+    # https://www.odoo.com/es_ES/forum/ayuda-1/question/how-to-add-a-field-conditional-to-tree-view-of-customer-14707
+    # play with context ?
+
     is_custom = fields.Boolean('Is custom value', help="Allow users to input custom values for this attribute value")
     html_color = fields.Char(
         string='HTML Color Index',
-        help="""Here you can set a
-        specific HTML color index (e.g. #ff0000) to display the color if the
-        attribute type is 'Color'.""")
+        help="""Here you can set a specific HTML color index (e.g. #ff0000) to display the color if the attribute type is 'Color'.""")
+    attribute_id = fields.Many2one('product.attribute')
+    display_type = fields.Selection(related='attribute_id.display_type', required=True)
```

Comment:
```suggestion
    display_type = fields.Selection(related='attribute_id.display_type', readonly=True)
```
`required` is inherited from attribute_id but `readonly=True` should always be set on related fields.